spiffworkflow-frontend/src/interfaces.ts

```diff
@@ -435,3 +435,35 @@ export interface DataStore {
   name: string;
   type: string;
 }
+
```

Comment:
i think these interfaces got moved to a different file.

Reply:
jason just committed to this branch killing them from this file.

Reply:
thank you